src/rabbit_shovel_parameters.erl

```diff
@@ -271,7 +271,10 @@ parse_dest(VHostName, ClusterName, Def, SourceHeaders) ->
 parse_amqp10_dest({_VHost, _Name}, _ClusterName, Def, SourceHeaders) ->
     Uris = get_uris(<<"dest-uri">>, Def),
     Address = pget(<<"dest-address">>, Def),
-    Properties = pget(<<"dest-properties">>, Def, []),
+    Properties = case pget(<<"dest-properties">>, Def, []) of
```

Comment:
This came up several times in the management plugin, so @michaelklishin added `rabbit_data_coercion:to_proplist`. I think you can use that here.